robotbase/src/main/java/com/team2357/frc2023/commands/ExtendArmToZeroCommand.java

```diff
@@ -0,0 +1,32 @@
+package com.team2357.frc2023.commands;
+
+import com.team2357.frc2023.subsystems.ArmExtensionSubsystem;
+import com.team2357.frc2023.Robot;
+import com.team2357.frc2023.Constants;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ExtendArmToZeroCommand extends CommandBase{
+    private static ArmExtensionSubsystem armExtensionSubsystem;
+    public ExtendArmToZeroCommand(){
+    addRequirements(Robot.m_armExtensionSubsystem);
```

Comment:
Why call this method and then set a local variable to Robot.m_armExtensionSubsystem. I would switch the order of these lines and initialize the local variable then use the `addRequirements(localVariable)` or get rid of the local variable completely.